CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.11.0 (2022-11-10)
```

Comment:
this should be 2.12.0

Reply:
Fixed here: [#ce3caf0](https://github.com/newrelic/nri-cassandra/pull/116/commits/ce3caf0cec74ace26f15a2182d9b1aa1916a5617)